README.md

```diff
@@ -291,16 +251,17 @@ Maybe you want to read:
 * [Config an experiment](docs/en_US/Tutorial/ExperimentConfig.md)
 * [How to use annotation](docs/en_US/TrialExample/Trials.md#nni-python-annotation)
 
+
 ## **Tutorials**
 
-* [Run an experiment on OpenPAI?](docs/en_US/PaiMode.md)
-* [Run an experiment on Kubeflow?](docs/en_US/KubeflowMode.md)
-* [Run an experiment on local (with multiple GPUs)?](docs/en_US/LocalMode.md)
-* [Run an experiment on multiple machines?](docs/en_US/RemoteMachineMode.md)
-* [Try different tuners](docs/en_US/tuners.rst)
-* [Try different assessors](docs/en_US/assessors.rst)
+* [Run an experiment on OpenPAI](docs/en_US/TrainingService/PaiMode.md)
+* [Run an experiment on Kubeflow](docs/en_US/TrainingService/KubeflowMode.md)
+* [Run an experiment on local (with multiple GPUs)](docs/en_US/TrainingService/LocalMode.md)
+* [Run an experiment on multiple machines](docs/en_US/TrainingService/RemoteMachineMode.md)
+* [Try different tuners](docs/en_US/Tuner/BuiltinTuner.md)
+* [Try different assessors](docs/en_US/Assessor/BuiltinAssessor.md)
 * [Implement a customized tuner](docs/en_US/Tuner/CustomizeTuner.md)
-* [Implement a customized assessor](docs/en_US/CustomizeAssessor.md)
+* [Implement a customized assessor](docs/en_US/Assessor/CustomizeAssessor.md)
 * [Use Genetic Algorithm to find good model architectures for Reading Comprehension task](examples/trials/ga_squad/README.md)
 
```

Comment:
Change this "examples/trials/ga_squad/README.md" to "" docs/en_US/TrialExample/SquadEvolutionExamples.md?